pyproject.toml

```diff
@@ -396,10 +396,12 @@ fix = true
 # don't delete print statements
 # don't remove unused imports
 # don't remove unused noqa
-unfixable = ["T201", "F401", "RUF100"]
```

Comment:
Nothing to change here right now, but if we have too many more ignored errors here, we'll probably want to have one item per line with inline comments

Reply:
Hm, let me remove the comments. I turned off `fix = true` for now because it's a bit too aggressive while developing. It's turned on for `pre-commit` though which I think is probably the appropriate time to auto fix things.

Reply:
Oh, whoops I misread this because the comment only shows up on the change line and no the addition line.